### PR TITLE
updated build.gradle in internal/logging and internal/runtime_shared so they support to install to local maven repository

### PR DESCRIPTION
**Pull request by ilovepumpkin**
I need to install dslink to local maven repository but after looking at the gradle build files and found it is not supported for local. So I updated them.

### CODE REVIEW

**Comment by Pacane** [comment]
Approve. @kaendfinger ?

File: internal/runtime_shared/build.gradle [hunk 1]
```diff
@@ -9,6 +9,17 @@ dependencies {
     compile 'com.jcraft:jzlib:1.1.3'
 }
 
+install {
+    repositories.mavenInstaller {
+        pom.project {
+                name = 'DSLink Shared Runtime'
+                artifactId = 'runtime_shared'
+                description = 'Shared Runtime for the SDK'
+            }
```
Comment (Pacane) [inline internal/runtime_shared/build.gradle]: Bad indent here.

File: internal/runtime_shared/build.gradle [hunk 2]
```diff
@@ -9,6 +9,17 @@ dependencies {
...
+    }
+}
+
+
```
Comment (Pacane) [inline internal/runtime_shared/build.gradle]: rm empty line

**Comment by kaendfinger** [comment]
Approve.

**Pull request merged**